apps/client/src/app/auth/_constants/constants/signUp/signUpUserType.array.ts

```diff
@@ -0,0 +1,12 @@
+export const CHOICE_USER_TYPE_GROUP = [
+  {
+    userType: "USER_BUYER",
```

Comment:
상수 이름이 `CHOICE_USER_TYPE_GROUP`로 명시되어 있으니
```suggestion
    type: "buyer",
```
으로 바꿔도 되지 않을까요?

Reply:
해당 값은 백앤드에서 지정해준 정보라서 소통 후 필요한 작업인듯 합니다.
우선 해당 리뷰는 패스하도록 하겠습니다.